docusaurus/docs/Android/04-compose/03-channel-components/03-channel-list.mdx

```diff
@@ -55,15 +55,15 @@ fun ChannelList(
     ),
     onLastItemReached: () -> Unit = { viewModel.loadMore() },
     onChannelClick: (Channel) -> Unit = {},
-    onChannelLongClick: (Channel) -> Unit = { viewModel.onChannelSelected(it) },
+    onChannelLongClick: (Channel) -> Unit = { viewModel.selectChannel(it) },
     ... // Other UI-related parameters
 )
 ```
 
-* `viewModel`: The instance of the `ChannelListViewModel` that this component reads data from and sends event to. Pass in your own instance if you want more control over your business logic, such as changing `Channel` filters or sort order in runtime.
-* `onLastItemReached`: Handler called when reaching the last item in the list, to trigger pagination. You don't need to override this if you're using the default `viewModel`, but if you're using a custom one, you can add custom behavior.
-* `onChannelClick`: Handler for the user tapping on an item. Useful for starting the `MessagesScreen` in your app.
-* `onChannelLongClick`: Handler for the user long tapping on an item. By default, this updates state in the `viewModel`, which you can read to show custom UI and `Channel` actions, if you're using a custom `ViewModel` instance. Override if you're using the default `viewModel` and you want to change the behavior.
+* `viewModel`: The instance of the `ChannelListViewModel` that this component reads data from and sends events to. Pass in your own instance if you want more control over your business logic, such as changing `Channel` filters or sort order in runtime.
+* `onLastItemReached`: Handler for the user reaching the last item in the list to trigger pagination. You don't need to override this if you're using the default `viewModel`, but if you're using a custom one, you can add custom behavior.
```

Comment:
Maybe a slightly different wording here would be better.

- for when the user reaches
- called when the user reaches

Reply:
or `Handler when the user reaches`